crates/ruff_linter/src/noqa.rs

```diff
@@ -1061,7 +1162,7 @@ mod tests {
         )];
 
         let contents = "x = 1";
-        let noqa_line_for = NoqaMapping::default();
+        let noqa_line_for: NoqaMapping = NoqaMapping::default();
```

Comment:
Is this necessary? (Is the type not inferred from the RHS?)

Reply:
Nope, not necessary 😅